src/test/ui/parser/block-no-opening-brace.rs

```diff
@@ -0,0 +1,30 @@
+// edition:2018
+
+#![feature(try_blocks)]
+
+fn main() {}
+
+fn f1() {
+   loop
+     let x = 0; //~ ERROR expected `{`, found keyword `let`
```

Comment:
How are multiple statements handled?

Reply:
I'll add another `let` to find out. :)

Reply:
Added another statement. Only the first statement is included.